### PR TITLE
#1435: add stats view for indexes

### PR DESCRIPTION
**Pull request by uweschaefer**

### CODE REVIEW

**Comment by uweschaefer** [comment]
closes #1435

File: factcast-store-pgsql/src/main/resources/db/changelog/factcast/issue1435/stats_index.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+       t.tablename,
+       c.reltuples::bigint                            AS num_rows,
+       pg_size_pretty(pg_relation_size(c.oid))        AS table_size,
+       psai.indexrelname                              AS index_name,
+       pg_size_pretty(pg_relation_size(i.indexrelid)) AS index_size,
+       CASE WHEN i.indisunique THEN 'Y' ELSE 'N' END  AS "unique",
+       psai.idx_scan                                  AS number_of_scans,
+       psai.idx_tup_read                              AS tuples_read,
+       psai.idx_tup_fetch                             AS tuples_fetched
+FROM pg_tables t
```
Comment (otbe) [inline factcast-store-pgsql/src/main/resources/db/changelog/factcast/issue1435/stats_index.sql:12]: Call me crazy but I would love to have `i.indisvalid` here as well 🥸

Comment (uweschaefer) [reply factcast-store-pgsql/src/main/resources/db/changelog/factcast/issue1435/stats_index.sql:22]: plz recheck

**Review by otbe — APPROVED** [review]
LGTM

**Pull request merged**